chains/iris/test.js

```diff
@@ -37,8 +37,8 @@ describe('CryPto iris test', function () {
         });
 
         it('test recover', function () {
-            let crypto = Irisnet.getCrypto(Irisnet.config.chain.iris,"testnet");
-            let account = crypto.recover("");
+            let crypto = Irisnet.getCrypto(Irisnet.config.chain.cosmos);
```

Comment:
Consider adding cosmos keys test in `chains/cosmos/test.js`?

Reply:
Iris and cosmos test cases are separate